src/openMVG/voctree/database.cpp

```diff
@@ -44,7 +44,7 @@ DocId Database::insert(DocId doc_id, const std::vector<Word>& document)
   return doc_id;
 }
 
-void Database::sanityCheck(size_t N, std::vector< std::vector<Match> >& matches) const
+void Database::sanityCheck(size_t N, std::map<size_t, DocMatches>& matches) const
```

Comment:
Is `size_t` in the map the ID of the view? I don't know how it works with the new UID, are they a `size_t` or `IndexT` or what?